src/common/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+
+import { ReactComponent as ArrowLeft } from '../../../assets/arrowLeft.svg';
+import { ReactComponent as ArrowRight } from '../../../assets/arrowRight.svg';
+
+import s from './Pagination.module.scss';
+
+export const Pagination = ({
+  onChange,
+  activePage,
+  totalPages,
+  arrowForwardOnClick,
+  arrowBackOnClick,
+}) => {
+  const [page, setValue] = useState(activePage);
+  useEffect(() => {
+    setValue(activePage);
```

Comment:
check it out, do you need the local state for pagination component

Reply:
no in wasn't needed so I deleted it